src/Tribe/Aggregator.php

```diff
@@ -58,8 +58,7 @@ class Tribe__Events__Aggregator {
 	 */
 	public static function instance() {
 		if ( ! isset( self::$instance ) ) {
```

Comment:
We are going to revisit these singleton methods in the future to kinda create a Std across the whole plugin suite. In this one it's up @borkweb to change. I would say that I like the ternary here because we are going to copy this across all classes, so it needs to be as small as possible, and behave the same way, that's why we don't declare a value also (PHP does it for us `null`).